keras/layers/advanced_activations.py

```diff
@@ -264,3 +264,55 @@ def get_config(self):
                   'a_right_init': self.a_right_init}
         base_config = super(SReLU, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+
+class MPELU(Layer):
+    '''Multiple Parametric Exponential Linear Units
+    `f(x) = alpha * (exp(x * beta) - 1) for x <= 0`,
+    `f(x) = x for x > 0`.
+
+    # Input shape
+        Arbitrary. Use the keyword argument `input_shape`
+        (tuple of integers, does not include the samples axis)
+        when using this layer as the first layer in a model.
+
+    # Output shape
+        Same shape as the input.
+
+    # Arguments
+        alpha_init: float. Initial value of the alpha weights.
+        beta_init: float. Initial values of the beta weights.
+        weights: initial weights, as a list of 2 numpy arrays.
+        
+    # References
+        - [Improving Deep Neural Network with Multiple Parametric Exponential Linear Units](http://arxiv.org/abs/1606.00305)
+    '''
+    def __init__(self, alpha_init=0.0, beta_init=1.0, weights=None, **kwargs):
+        self.supports_masking = True
+        self.alpha_init = K.cast_to_floatx(alpha_init)
+        self.beta_init = K.cast_to_floatx(beta_init)
+        self.initial_weights = weights
+        super(MPELU, self).__init__(**kwargs)
+
+    def build(self, input_shape):
+        input_shape = input_shape[1:]
+        self.alphas = K.variable(self.alpha_init * np.ones(input_shape),
+                                 name='{}_alphas'.format(self.name))
+        self.betas = K.variable(self.beta_init * np.ones(input_shape),
+                                 name='{}_betas'.format(self.name))
+        self.trainable_weights = [self.alphas, self.betas]
+
+        if self.initial_weights is not None:
+            self.set_weights(self.initial_weights)
+            del self.initial_weights
+
+    def call(self, x, mask=None):
+        pos = K.relu(x)
+        neg = self.alphas * (K.exp((x - abs(x)) * 0.5 * self.betas) - 1)
```

Comment:
@xingdi-eric-yuan haha I haven't realized that until your comment. Tried to google that just now, maybe it's because of this:

![3](https://cloud.githubusercontent.com/assets/1331389/15760126/0d85a8ea-293c-11e6-9695-0979cad4606d.PNG)